examples/electron/README.md

```diff
@@ -11,7 +11,7 @@ directory to ensure assets used from examples/web are also within the scope of t
 node process.
 
 <blockquote>
-   [examples/web] $ electron ../electron
+   [examples/web] $ electron ../electron --enable-logging
```

Comment:
This is included in this commit because it allowed viewing of the example ipc subscribe from the main Electron process outputing to console (see electron.js bootstrap changes).